optimade/server/mappers/entries.py

```diff
@@ -363,6 +363,10 @@ def map_back(cls, doc: dict) -> dict:
     def deserialize(
         cls, results: Union[dict, Iterable[dict]]
     ) -> Union[List[EntryResource], EntryResource]:
+        """Converts the raw database entries for this class into serialized models,
+        mapping the data along the way.
+
+        """
         if isinstance(results, dict):
             return cls.ENTRY_RESOURCE_CLASS(**cls.map_back(results))
```

Comment:
I think these lines are no longer needed for our implementation, now we always pass a list.
```suggestion
    def deserialize(
        cls, results: Iterable[dict]
    ) -> List[EntryResource]:
        """Converts the raw database entries for this class into serialized models,
        mapping the data along the way.
        """
```